frappe/website/doctype/website_theme/website_theme_template.scss

```diff
@@ -16,7 +17,12 @@ $enable-rounded: {{ button_rounded_corners and "true" or "false" }};
 // Bootstrap Variable Overrides
 {{ custom_overrides or '' }}
 
-@import "frappe/public/scss/website";
+// Import themes from installed apps
+{% for import_path in website_theme_scss -%}
```

Comment:
@barredterra 

Instead of importing scss files from hooks, I think we should import "{app}/public/scss/website" from each app and ignore if it's not available. This follows the "Convention over configuration" approach that is used for web pages and templates.

Also, I don't like the UX of "imports_to_ignore".

Reply:
Yes, i can switch to the "convention" approach you're proposing.

It's definetely necessary to ignore core styles. This is one of the main reasons for this PR. For example, I have a bootstrap theme that covers all use cases but it does not override every single frappe style, which messes things up.